src/reanimated2/Hooks.js

```diff
@@ -109,7 +109,6 @@ function runAnimations(animation, timestamp, key, result) {
       animation.timestamp = timestamp;
       if (finished) {
         animation.finished = true;
-        // TODO: also trigger callback when animation is cancelled (overwritten)
```

Comment:
Has it been resolved?

Reply:
In fact I checked all the animations and that does not work for all of them, good point, reverting

Reply:
ok so I took a deeper look and it does not seem to be a case anymore. Triggering a callback on animation cancel works for:
- withTiming
- withSpring
- withDecay
- delay
- sequence
- repeat
so I think I covered all the animations for now